pkg/registry/apis/alerting/notifications/receiver/conversions.go

```diff
@@ -58,7 +58,8 @@ func convertToK8sResource(
 	namespacer request.NamespaceMapper,
 ) (*model.Receiver, error) {
 	spec := model.Spec{
-		Title: receiver.Name,
+		Title:        receiver.Name,
+		Integrations: []model.Integration{},
```

Comment:
NIT: Since we have exact knowledge we can pre-size the slice
```suggestion
		Integrations: make([]model.Integration, 0, len(receiver.Integrations)),
```